src/main/java/frc/robot/maps/Shrimp.java

```diff
@@ -0,0 +1,171 @@
+        motor.getMotorController().setPeriodicFramePeriod(PeriodicFrame.kStatus2, status2);
+    }
+
+    @Override
+    public SwerveDriveMap getDriveMap() {
+
+        // Remember to divide by 360
+        // CAN ID 2
+        final double FLOFFSET = 0;
+        // CAN ID 4
+        final double FROFFSET = 0;
+        // CAN ID 1
+        final double RLOFFSET = 0;
+        // CAN ID 3
+        final double RROFFSET = 0;
+
+        // Value taken from CAD as offset from center of module base pulley to center
+        // of the robot
+        final double MODULE_OFFSET_XY = Units.inchesToMeters(6.00015);
+        final PigeonGyro2 pigeonGyro2 = new PigeonGyro2(1);
+
+        final CSSparkMax frontLeftSteer = new CSSparkMax(4);
+        final CSSparkMax frontRightSteer = new CSSparkMax(8);
+        final CSSparkMax rearLeftSteer = new CSSparkMax(2);
+        final CSSparkMax rearRightSteer = new CSSparkMax(6);
+
+        CSSparkMax frontLeftDrive = new CSSparkMax(3);
+        CSSparkMax frontRightDrive = new CSSparkMax(7);
+        CSSparkMax rearLeftDrive = new CSSparkMax(1);
+        CSSparkMax rearRightDrive = new CSSparkMax(5);
+
+        setStatusPeriods(frontLeftSteer, 100, 100, 100);
+        setStatusPeriods(frontRightSteer, 100, 100, 100);
+        setStatusPeriods(rearLeftSteer, 100, 100, 100);
+        setStatusPeriods(rearRightSteer, 100, 100, 100);
+
+        setStatusPeriods(frontLeftDrive, 100, 10, 10);
+        setStatusPeriods(frontRightDrive, 100, 10, 10);
+        setStatusPeriods(rearLeftDrive, 100, 10, 10);
+        setStatusPeriods(rearRightDrive, 100, 10, 10);
+
+        frontLeftSteer.getMotorController().setInverted(false);
+        frontRightSteer.getMotorController().setInverted(false);
+        rearLeftSteer.getMotorController().setInverted(false);
+        rearRightSteer.getMotorController().setInverted(false);
+
+        frontLeftDrive.setInverted(true);
+        frontLeftDrive.getEncoder().getRaw().setMeasurementPeriod(8);
+        frontLeftDrive.getEncoder().getRaw().setAverageDepth(2);
+        frontRightDrive.setInverted(false);
+        frontRightDrive.getEncoder().getRaw().setMeasurementPeriod(8);
+        frontRightDrive.getEncoder().getRaw().setAverageDepth(2);
+        rearLeftDrive.setInverted(true);
+        rearLeftDrive.getEncoder().getRaw().setMeasurementPeriod(8);
+        rearLeftDrive.getEncoder().getRaw().setAverageDepth(2);
+        rearRightDrive.setInverted(false);
+        rearRightDrive.getEncoder().getRaw().setMeasurementPeriod(8);
+        rearRightDrive.getEncoder().getRaw().setAverageDepth(2);
+
+        frontLeftSteer.getMotorController().setSmartCurrentLimit(30);
+        frontRightSteer.getMotorController().setSmartCurrentLimit(30);
+        rearLeftSteer.getMotorController().setSmartCurrentLimit(30);
+        rearRightSteer.getMotorController().setSmartCurrentLimit(30);
+
+        // Configuration for MK4i with L2 speeds
```

Comment:
Shrimp is going to have MK4 modules not MK4i. Still L2 speeds.
This will also impact the motor inverting since the motors are mounted the other direction on the MK4.